package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "server-training",
+  "version": "1.0.0",
+  "description": "\"# ServerTraining\"",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "tsc": "tsc",
+    "start:dev": "nodemon --config nodemon.json src/app.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lorenpap/ServerTraining.git"
+  },
+  "bugs": {
+    "url": "https://github.com/lorenpap/ServerTraining/issues"
+  },
+  "homepage": "https://github.com/lorenpap/ServerTraining#readme",
+  "dependencies": {
+    "@types/node": "^14.14.14",
```

Comment:
move @types/node and typescript to devDependencies AND "nodemon": "^2.0.4",
    "ts-node": "^8.10.2" to Dependencies